lib/Conversion/TorchToTcp/TcpCustomOp.cpp

```diff
@@ -146,6 +147,89 @@ class ConvertAten_IndexPutImplOp
   }
 };
 
+class ConvertAtenConvolutionOp : public OpConversionPattern<AtenConvolutionOp> {
+public:
+  using OpConversionPattern<AtenConvolutionOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(AtenConvolutionOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    SmallVector<Type> resultTypes;
+    if (failed(OpConversionPattern<AtenConvolutionOp>::getTypeConverter()
+                   ->convertTypes(op->getResultTypes(), resultTypes))) {
+      return failure();
+    }
+
+    SmallVector<Value> operands;
+    SmallVector<StringRef> operandNames;
+
+    auto addOperand = [&](std::string name, Value value) {
+      operandNames.push_back(name);
+      operands.push_back(value);
+    };
```

Comment:
Non-blocking and could be addressed in a follow-on PR:
This would be nice to factor out as a utility, so they can be reused by other TcpCustomOp conversion patterns.